script/Ciel/Day 5/05_01_CIEL05_1B.txt

```diff
@@ -11,7 +11,7 @@
 [sha:b1956eca42a00fff40877026369bf3e8a74b2f61]{
 -- Page 10002, Offset 11021.
 -- 「いいじゃんよぅトシ！　多い分にはいいんだからさァ！
-"Eh, it's all good Toshi! The more the merrier after all!
+"Huh, it's all good Toshi! The more the merrier after all!
```

Comment:
Naw, it's all good Toshi!

Reply:
i'm not even sure where this one came from

Reply:
it's a Ciel file

Reply:
no... i see what file it is, I'm just saying I don't know where in the source text "Eh," came from. Transliterated it's "ii jan yo toshi"